app/controllers/admin/users_controller.rb

```diff
@@ -29,6 +29,11 @@ def destroy
     redirect_to action: :index
   end
 
+  def resend_email
+    @user = User.find(params[:id])
+    @user.resend_confirmation_instructions
+  end
```

Comment:
I had been making the link available as quite often I was able to contact people directly to share the link when the email just didn't seem to be working. The functionality to resend the email address already exists here: https://members.londondecom.org/users/confirmation/new

Reply:
Yes, it does exist there, but it gives us the option to send the user there, or force send it ourselves if they are struggling, better to have two ways I feel

Reply:
I'll remove the feature then. I did try setup something to redirect to it complete with the data in the form but it was never happy to have the data passed to the form so I took the cleaner approach.